pkg/gofr/gofr_test.go

```diff
@@ -321,41 +321,24 @@ func Test_AddRESTHandlers(t *testing.T) {
 }
 
 func Test_initTracer(t *testing.T) {
-	mockConfig1 := config.NewMockConfig(map[string]string{
-		"TRACE_EXPORTER": "zipkin",
-		"TRACER_URL":     "http://localhost:2005/api/v2/spans",
-	})
-
-	mockConfig2 := config.NewMockConfig(map[string]string{
-		"TRACE_EXPORTER": "jaeger",
-		"TRACER_URL":     "localhost:2005",
-	})
-
-	mockConfig3 := config.NewMockConfig(map[string]string{
-		"TRACE_EXPORTER": "gofr",
-	})
-
-	mockConfig4 := config.NewMockConfig(map[string]string{
-		"TRACE_EXPORTER":  "zipkin",
-		"TRACER_URL":      "http://localhost:2005/api/v2/spans",
-		"TRACER_AUTH_KEY": "valid-token",
-	})
-
-	mockConfig5 := config.NewMockConfig(map[string]string{
-		"TRACE_EXPORTER":  "jaeger",
-		"TRACER_URL":      "localhost:2005",
-		"TRACER_AUTH_KEY": "valid-token",
-	})
-
-	mockConfig6 := config.NewMockConfig(map[string]string{
-		"TRACE_EXPORTER": "zipkin",
-		"TRACER_URL":     "https://tracer-service.dev",
-	})
-
-	mockConfig7 := config.NewMockConfig(map[string]string{
-		"TRACE_EXPORTER": "jaeger",
-		"TRACER_URL":     "https://tracer-service.dev",
-	})
+	createMockConfig := func(traceExporter, host, port, url, authKey string) config.Config {
+		return config.NewMockConfig(map[string]string{
+			"TRACE_EXPORTER":  traceExporter,
+			"TRACER_HOST":     host,
+			"TRACER_PORT":     port,
+			"TRACER_URL":      url,
+			"TRACER_AUTH_KEY": authKey,
+		})
+	}
+	mockConfig1 := createMockConfig("zipkin", "localhost", "2005", "", "")
+	mockConfig2 := createMockConfig("jaeger", "localhost", "2005", "", "")
+	mockConfig3 := createMockConfig("gofr", "", "", "", "")
+	mockConfig4 := createMockConfig("zipkin", "localhost", "2005", "", "valid-token")
+	mockConfig5 := createMockConfig("jaeger", "localhost", "2005", "", "valid-token")
+	mockConfig6 := createMockConfig("zipkin", "", "", "https://tracer-service.dev", "")
+	mockConfig7 := createMockConfig("jaeger", "", "", "https://tracer-service.dev", "")
+	mockConfig8 := createMockConfig("otlp", "localhost", "2005", "", "")
```

Comment:
I'm unsure why old host and port are still present in the test.

I would have removed them completely

Reply:
I'm sorry, i forget it